src/Chunk.h

```diff
@@ -82,6 +82,14 @@ class cChunk :
 		cpPresent,  /**< The chunk is present */
 	};
 
+	enum class eSaveStatus
+	{
+		CLEAN,                 // No changes were made to the chunk since last save
```

Comment:
Yes please!
Also, we mostly use Delphi-like prefixes for enum values, so these should be `ssClean`, `ssDirty` etc. See the `ePresence` values above.